src/pages/resume.js

```diff
@@ -45,270 +48,145 @@ const Resume = () => (
-            their new year media &amp; print campaigns. The site was built on
-            NodeJS &amp; included Salesforce integration for CRM and Google Maps
-            integration for an interactive location finder of Nuffield Hospitals
-            by postcode.
+          As part of my university course, I worked on an Internet of Things project, 
+          to design a smart city software solution for Belfast, with the aim of lowering traffic levels.
+
+          I conducted research and analysis to provide a focused solution 
+          to the traffic problem that exists in Belfast City.
+
+          The solution consisted of a full-stack application, 
+          in which the server was implemented in Python, and the front-end relied on HTML, CSS and JavaScript.
           </p>
         </div>
       </section>
       <section className="entry-list__item row">
         <div className="column">
           <span className="entry-list__meta">
-            <time datetime="2013-07-28">Jul '13</time> &ndash;
-            <time datetime="2014-11-28">Nov '14 (Long-term contract)</time>
+            <time datetime="2014-12-01">Sept '16</time> &ndash;
+            <time datetime="2015-01-28">Sept '17 (MSc in Software Development)</time>
           </span>
         </div>
         <div className="column column-75">
           <h3 className="entry-list__heading">
-            <img src={futurelearnLogo} alt="FutureLearn logo" />
+            <img src={qubLogo} alt="Qub logo" />
           </h3>
           <p>
-            I joined a small team of developers four months after{' '}
-            <em>FutureLearn</em> was founded to help build a beta product fit
-            for public unveiling in mid-Sept.
-          </p>
-          <p>
-            Within the first 3 months, the user base quickly grew to over
-            250,000 registered users. In Oct 2014, after just one year live,
-            FutureLearn is fast approaching 1.5 million enrolments.
-          </p>
-          <p>
-            The product is built in Ruby on Rails, using HAML and SASS on the
-            front end. The team worked using Agile to gain real-time feedback
-            from beta users to shape the product. I was one of two contracting
-            Front Enders who helped build a scalable, modular, front end
-            architecture which the maturing team can now inherit and build upon
-            further.
+          <a href="https://www.qub.ac.uk/courses/postgraduate-taught/software-development-msc/"></a>
+          During the academic year of 2016/17 I enrolled in the MSc Software Development
+          course offered by Queen's University Belfast.
+
+          This course gave me an opportunity to analyse and solve problems 
+          through the design of robust and elegant software systems. 
+          Skills developed included Java programming, web development, 
+          database design, and software testing.
```

Comment:
You should add a link to your blog post at the end of this section.

Reply:
Fair point ... I was trying to do this at the start ... but unfortunately, I was unsuccessful ...